### PR TITLE
Image-zoom

### PR DESCRIPTION
**Pull request by nileshgulia1**
#647 
-cc @tisto

### CODE REVIEW

File: src/components/manage/Tiles/Image/__snapshots__/View.test.jsx.snap [hunk 1]
```diff
@@ -6,8 +6,18 @@ exports[`renders a view image component 1`] = `
 >
   <img
     alt=""
-    className="ui image"
```
Comment (tisto) [inline src/components/manage/Tiles/Image/__snapshots__/View.test.jsx.snap]: @nileshgulia1 why did you remove the semantic ui class here? That breaks the image handling in Volto.

Comment (nileshgulia1) [reply src/components/manage/Tiles/Image/__snapshots__/View.test.jsx.snap]: will undo the snapshots....

Comment (tisto) [reply src/components/manage/Tiles/Image/__snapshots__/View.test.jsx.snap]: @nileshgulia1 no! this is no solution. You broke Volto by taking away the semantic ui classes. You have to amend the "<ImageZoom" to implement the same semantic ui classes that were there before.

Comment (nileshgulia1) [reply src/components/manage/Tiles/Image/__snapshots__/View.test.jsx.snap]: @tisto <ImageZoom/> by default overrides css which we use. We have replaced our <Image/> from semantic ui which contains css classes already. The workaround could be just adding the css manually which semantic ui provides. 
https://github.com/Semantic-Org/Semantic-UI-React/blob/d6f20f90f35494a2eeeb0c1586658172fe98ea06/src/elements/Image/Image.js#L54

Comment (tisto) [reply src/components/manage/Tiles/Image/__snapshots__/View.test.jsx.snap]: You have to pass those css classes to the ImageZoom component.

**Comment by tisto** [comment]
@sneridagh works like a charm and looks good to me. Would you mind having a second look?
